inc/compat/gutenberg.php

```diff
@@ -16,24 +16,32 @@
  */
 function primer_gutenberg_theme_support() {
 
+	// Load default block styles.
 	add_theme_support( 'wp-block-styles' );
+
+	// Add support for full and wide alignment.
 	add_theme_support( 'align-wide' );
 
+	// Enqueue editor styles if the block editor exists.
+	if ( function_exists( 'register_block_type' ) ) {
+
+		$suffix = SCRIPT_DEBUG ? '' : '.min';
+
+		add_theme_support( 'editor-styles' );
+
+		// Load regular editor styles into the new block-based editor.
+		add_editor_style( "assets/css/admin/editor-blocks{$suffix}.css" );
+	}
+
 }
 add_action( 'after_setup_theme', 'primer_gutenberg_theme_support' );
 
 /**
- * Enqueue styles for Gutenberg editor.
- *
- * @since 1.8.5
+ * Enqueue supplemental block editor styles.
  */
-function primer_gutenberg_editor_assets() {
-
-	global $post;
+function primer_editor_frame_styles() {
 
 	$suffix = SCRIPT_DEBUG ? '' : '.min';
-
-	wp_enqueue_style( 'primer-gutenberg-editor', get_template_directory_uri() . "/assets/css/admin/gutenberg-editor{$suffix}.css", true, defined( 'PRIMER_CHILD_VERSION' ) ? PRIMER_CHILD_VERSION : PRIMER_VERSION, 'all' );
-
+	wp_enqueue_style( Primer_Customizer::$stylesheet . '-editor-frame', get_template_directory_uri() . "/assets/css/admin/editor-frame{$suffix}.css", array(), PRIMER_VERSION, 'all' );
```

Comment:
@richtabor Add a space here between `$suffix` and `wp_enqueue_style` and then again between `wp_enqueue_style` and the closing `}` of the function.